src/Documenter.jl

```diff
@@ -405,13 +405,12 @@ function deploydocs(;
     - ENV["TRAVIS_REPO_SLUG"]="$(travis_repo_slug)" occurs in repo="$(repo)" $(marker(repo_ok))
     - ENV["TRAVIS_PULL_REQUEST"]="$(travis_pull_request)" is "false" $(marker(pr_ok))
     - ENV["TRAVIS_TAG"]="$(travis_tag)" is (i) empty or (ii) a valid VersionNumber $(marker(tag_ok))
-    - ENV["TRAVIS_BRANCH"]="$(travis_branch)" matches devbranch="$(devbranch)" $(marker(branch_ok))
+    - ENV["TRAVIS_BRANCH"]="$(travis_branch)" matches devbranch="$(devbranch) (if tag is empty) $(marker(branch_ok))
     - ENV["DOCUMENTER_KEY"] exists $(marker(key_ok))
     Deploying: $(marker(should_deploy))
```

Comment:
Completely unnecessary bikeshedding thing -- I started wondering if it might be better to have the markers before the string, e.g.

```
┌ Debug: Deployment criteria:
│ - ✔ ENV["TRAVIS_REPO_SLUG"]="JuliaLang/julia" occurs in repo="JuliaLang/julia"
│ - ✔ ENV["TRAVIS_PULL_REQUEST"]="false" is "false"
│ - ✔ ENV["TRAVIS_TAG"]="v0" is (i) empty or (ii) a valid VersionNumber
│ - ✔ ENV["TRAVIS_BRANCH"]="master" matches devbranch="master"
│ - ✔ ENV["DOCUMENTER_KEY"] exists
│ Deploying: ✔
└ @ Main REPL[32]:1
```